src/ZeekString.cc

```diff
@@ -352,8 +352,8 @@ String::Vec* String::VecFromPolicy(VectorVal* vec) {
 char* String::VecToString(const Vec* vec) {
```

Comment:
Just because I looked, these three methods on String appear to never be called within the tree, should we deprecate them?
```
    // Helper functions for vectors:
    static VectorVal* VecToPolicy(Vec* vec);
    static Vec* VecFromPolicy(VectorVal* vec);
    static char* VecToString(const Vec* vec);
```